x-pack/metricbeat/module/iis/application_pool/_meta/fields.yml

```diff
@@ -1,10 +1,76 @@
 - name: application_pool
   type: group
-  release: beta
+  release: ga
   description: >
     application_pool
   fields:
     - name: name
       type: keyword
       description: >
         application pool name
+    - name: process
+      type: group
+      description: >
+        process details
+      fields:
+        - name: pid
```

Comment:
Is this equivalent to the pid of the operating system? If it is, this should be probably stored in the ECS `process.pid` field, and better to do the change before going GA :slightly_smiling_face: 
If it isn't the same, we could add something about this in the description.

Reply:
the field is equivalent to the pid and has been mapped as such in the code, I have removed the mapping inside the fields.yml file.